modules/common/src/Util/DrupalFiles.php

```diff
@@ -98,7 +123,65 @@ public function retrieveFile($url, $destination) {
       return $this->fileCreateUrl("{$destination}/{$filename}");
     }
     // Handle http(s):// URIs.
-    return system_retrieve_file($url, $destination, FALSE, FileSystemInterface::EXISTS_REPLACE);
+    return $this->retrieveRemoteFile($url, $destination);
```

Comment:
Because this is the only call to `system_retrieve_file()`, and we always pass it `false` for managed and replace for file exists, we don't need to support parameters for this. Meaning we don't need to have any support at all for managed files in this class.